tests/functional/test_regions.py

```diff
@@ -546,3 +546,17 @@ def test_unregionalized_client_with_unknown_region(self):
         self.assertTrue(
             stubber.requests[0].url.startswith('https://iam.amazonaws.com/')
         )
+
+
+@pytest.mark.parametrize("is_builtin", [(True,), (False,)])
```

Comment:
pretty sure you can just pass as a list if there is just one parameter, it doesn't need to be tuples with a single value.

```suggestion
@pytest.mark.parametrize("is_builtin", [True, False])
```

Reply:
Making that change makes the test fail with:

```
TypeError: TestLoader.test_load_data_optionally_returns_path() takes 1 positional argument but 2 were given
```

Reply:
Changed in 7aeb81341